metau-capstone/View Controllers/ImportViewController.m

```diff
@@ -0,0 +1,91 @@
+//
+//  ImportViewController.m
+//  metau-capstone
+//
+//  Created by Shirley Zhu on 7/21/22.
+//
+
+#import "ImportViewController.h"
+#import "APIManager.h"
+
+@interface ImportViewController ()
+
+@end
+
+@implementation ImportViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+- (IBAction)didTapSubmitImport:(UIButton *)sender {
+    NSString *pathParameters = [self pathParameters];
+    if (![pathParameters isEqualToString:@"invalid"]) {
+        [[APIManager shared] getSheetsData:pathParameters withCompletetion:^(NSError *error) {
+            if (!error) {
+                NSLog(@"get request");
+                UIAlertController * alert = [UIAlertController alertControllerWithTitle:@"Success ^-^ !!" message:@"Your flashcards were created." preferredStyle:(UIAlertControllerStyleAlert)];
+                UIAlertAction * okAction = [UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault handler:^(UIAlertAction * _Nonnull action) {
+                }];
+                [alert addAction:okAction];
+                [self presentViewController:alert animated:YES completion:^{}];
```

Comment:
This may cause an issue related to retain cycle when network call is involved. you can read more about this https://fburl.com/wiki/i1tirtjk, and this wiki also contain the recommended pattern going forward